common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js

```diff
@@ -0,0 +1,24 @@
+try {
+    ((document, window) => {
+        var ad = document.createElement('div');
+        ad.style.position = 'absolute';
+        ad.style.left = '-9999px';
```

Comment:
this bugs me - will try a switch if this merges to use a different method to hide it. i'm curious as to whether it'll [affect iOS crashes](http://www.zeldman.com/2012/03/01/replacing-the-9999px-hack-new-image-replacement/)